astrowidgets/_astropy_init.py

```diff
@@ -17,11 +17,6 @@
     from .version import version as __version__
 except ImportError:
     __version__ = ''
-try:
-    from .version import githash as __githash__
-except ImportError:
-    __githash__ = ''
-
 
 if not _ASTROPY_SETUP_:  # noqa
```

Comment:
Don't need `_ASTROPY_SETUP_` no more throughout this file. As a result, you can also dedent the config stuff.